apps/settings/elements/developer_service_workers.html

```diff
@@ -0,0 +1,23 @@
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+
+<element name="developer-service-workers" extends="section">
+  <template>
+
+    <gaia-header action="back" data-href="#developer">
+      <h1>Service Workers</h1>
+    </gaia-header>
+
+    <section id="service-workers-list-section">
```

Comment:
I did not notice this in the last review. `section` in settings app represents a panel, so the height and width are limited to 100%. Please change this to `div`.